src/main/pages/setup-docker/docker-config.adoc

```diff
@@ -1,54 +1,55 @@
 ---
-title: "Configuration&#58 Docker"
+title: "Configuring Che on Docker"
 keywords: docker, configuration
 tags: [installation, docker]
 sidebar: user_sidebar
 permalink: docker-config.html
 folder: setup-docker
 ---
 
+The Che on Docker configuration is handled by modifying the https://github.com/eclipse/che/blob/master/dockerfiles/init/manifests/che.env[che.env] file that is placed in the root directory of a host folder volume mounted to `:/data`. This configuration file is generated during the `che init` phase. If you rerun the `che init` command in an already initialized folder, the process will abort unless you pass `--force`, `--pull`, or `--reinit`.
 
-Configuration is handled by modifying https://github.com/eclipse/che/blob/master/dockerfiles/init/manifests/che.env[che.env] placed in the root of a host folder volume mounted to `:/data`. This configuration file is generated during the `che init` phase. If you rerun `che init` in an already initialized folder, the process will abort unless you pass `--force`, `--pull`, or `--reinit`.
+You can also pass an environment variable directly in the docker run syntax: `-e CHE_ENV_NAME=value`.
```

Comment:
This should either be "Docker run" or "`docker run`". (I think `docker run` makes the most sense.)